### DISCUSSION TITLE
Java 8 Compatibility:The CheckBox in the MFXCheckListView is not displayed until it is clicked

### DISCUSSION THREAD

**Discussion by burningtnt** [palexdev/MaterialFX#360] [Q&A]
Glavo and I are trying to make MaterialFX compatible with Java 8 [here](https://github.com/Glavo/MaterialFX-Java8). By now, most of the bugs have been resolved except [this one](https://github.com/Glavo/MaterialFX-Java8/issues/3). Do any body know why it happened?

Here is the raw content of that issue.

> #### When running on Java8, the CheckBox in the MFXCheckListView is not displayed until it is clicked.
> ![image](https://github.com/palexdev/MaterialFX/assets/88144530/3b2d08de-4df8-4e18-8156-35144ef0896c)
> After clicking a CheckBox, it will be displayed:
> ![image](https://github.com/palexdev/MaterialFX/assets/88144530/f00ce216-267c-4a07-829b-e936cab2a373)
> Then uncheck it again, it can still be displayed normally:
> ![image](https://github.com/palexdev/MaterialFX/assets/88144530/0262571b-fedd-4f7c-b671-c2d2202aeea5)

**Comment by palexdev** [comment]
The checkbox skin has been slightly changed in a "recent" commit, it could be that or the culprit could be CSS
In any case you could use ScenicView to check what's going on

**Comment by burningtnt** [comment]
Thanks, I checked it and found the problem is caused by the un-initialized node clip. Thank you again~
